src/adaptors/abracadabra/cauldrons.js

```diff
@@ -551,9 +571,9 @@ const marketInfoToPool = (chain, marketInfo, collateral, pricesObj) => {
   const ltv = marketInfo.maximumCollateralRatio / 10000;
 
   const pool = {
-    pool: `${marketInfo.cauldron}-${chain}`,
+    pool: `${marketInfo.address}-${chain}`,
```

Comment:
is this change necessary? if not, better to keep what we have in master

Reply:
i see. the addresses aren't the same right? the problem is that whenever we update pool values, i need to manually update the db to the new pool values. otherwise we'll loose the reference to the pools historic values we have collected so far

Reply:
Yes, the value is exactly the same. It's just a different "variable"/field name.